.github/workflows/build-m1-binaries.yml

```diff
@@ -123,12 +133,19 @@ jobs:
           . packaging/pkg_helpers.bash
 
           if [[ $CHANNEL == "test" ]]; then
+            setup_cuda
             setup_base_build_version
+            setup_macos
           else
+            setup_cuda
```

Comment:
I don't think there are any plans for M1+CUDA unless you know something nobody else does :)